SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracerUserInterface.cpp

```diff
@@ -11,6 +11,27 @@
 
 DEFINE_LOG_CATEGORY(LogSpatialEventTracerUserInterface);
 
+namespace
+{
+TArray<FSpatialGDKSpanId> ConvertSpanIds(const TArray<FUserSpanId>& Causes)
+{
+	TArray<FSpatialGDKSpanId> CauseSpanIds;
+	for (const FUserSpanId& UserSpanIdCause : Causes)
+	{
+		if (!UserSpanIdCause.IsValid())
+		{
+			UE_LOG(LogSpatialEventTracerUserInterface, Warning,
+				   TEXT("USpatialEventTracerUserInterface::CreateSpanIdWithCauses - Invalid input cause"));
+			continue;
+		}
+
+		FSpatialGDKSpanId CauseSpanId = SpatialGDK::SpatialEventTracer::UserSpanIdToGDKSpanId(UserSpanIdCause);
```

Comment:
This defo cant fail?

Reply:
Nah, only if `¬UserSpanIdCause.IsValid` but that case is already checked above ^